task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,18 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if (num % 3 == 0 && num % 5 !== 0) {
+        return 'Fizz';
+    }
+    else if (num % 3 !== 0 && num % 5 == 0) {
```

Comment:
since the block above has a return statement you can omit `else `and just use plain `if `statement